src/controllers/v1/health/index.ts

```diff
@@ -2,9 +2,9 @@ import { Controller, Get, Route, Request } from 'tsoa'
 import { Logger } from 'pino'
 
 import * as Ex from 'express'
-import type { Health } from '../../models'
-import { logger } from '../../lib/logger'
-@Route('health')
+import type { Health } from '../../../models'
+import { logger } from '../../../lib/logger'
+@Route('v1/health')
```

Comment:
previously we haven't prefixed the `/health` endpoint with version e.g. in `dscp-identity-service` https://github.com/digicatapult/dscp-identity-service/blob/89e616ed8cb526412bb145f27786646df11cf59e/app/server.js#L51

just double checking we want to move health as well?

Reply:
agreed offline to leave the health endpoint at `/health`